pdr_backend/pdr_dashboard/test/test_util_format.py

```diff
@@ -0,0 +1,80 @@
+from pdr_backend.pdr_dashboard.util.format import (
+    format_eth_address,
+    format_currency,
+    format_percentage,
+    format_currency_with_decimal,
+    format_currency_with_decimal_and_suffix,
+    format_avg_currency_with_decimal,
+    format_value,
+)
+
+
+def test_format_eth_address():
+    assert (
+        format_eth_address("0x1234567890123456789012345678901234567890")
+        == "0x123...67890"
+    )
+    assert (
+        format_eth_address("0x123456789012345678901234567890123456789")
+        == "0x123...56789"
+    )
+
+
+def test_format_eth_address_empty():
+    assert format_eth_address("") == "..."
```

Comment:
Why `...` is returned when the address is empty? It should return an empty string or return a message such as: 'No address'